lms/envs/common.py

```diff
@@ -1860,6 +1860,9 @@
     # Microsite configuration
     'microsite_configuration',
 
+    # RSS Proxy
+    'rss_proxy',
```

Comment:
Yes, I think lms is the right place for now, since this is where marketing/onboarding views are located. I think ultimately the rss_proxy should be moved along with marketing/onboarding.